### PR TITLE
known_issues: Updated known issues for Asset Tracker v2

### PR DESCRIPTION
**Pull request by simensrostad**
Updated known issues for Asset Tracker v2

### CODE REVIEW

**Comment by simensrostad** [comment]
@greg-fer This commits adds a new section that separates known issues for the Asset Tracker versions.
@jhn-nordic @jtguggedal Is there any of the sections under Asset Tracker that also applies for version 2?


Same question here.

File: doc/nrf/known_issues.rst [hunk 1]
```diff
@@ -112,6 +107,21 @@ IRIS-2676: Missing support for FOTA on nRF Cloud
...
 					* f520159f0415f011ae66efb816384a8f7bade83d
 
+Asset Tracker v2
+================
+
+.. rst-class:: v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
+
+CIA-351: Connectivity issues with Azure IoT Hub
+  If a c2d message is sent to the device while the device is in LTE Power Saving Mode, the TCP connection will most likely be terminated by the server.
+  Known symptoms of this is frequent reconnections to cloud, that messages sent to Azure IoT Hub never arrives, and that FOTA images are downloaded twice.
```
Comment (greg-fer) [inline doc/nrf/known_issues.rst]:
```suggestion
  Known symptoms of this are frequent reconnections to cloud, messages sent to Azure IoT Hub never arriving, and FOTA images being downloaded twice.
```

File: doc/nrf/known_issues.rst [hunk 2]
```diff
@@ -112,6 +107,21 @@ IRIS-2676: Missing support for FOTA on nRF Cloud
 					* 156d4cf3a568869adca445d43a786d819ae10250
 					* f520159f0415f011ae66efb816384a8f7bade83d
 
+Asset Tracker v2
+================
+
+.. rst-class:: v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
+
+CIA-351: Connectivity issues with Azure IoT Hub
+  If a c2d message is sent to the device while the device is in LTE Power Saving Mode, the TCP connection will most likely be terminated by the server.
```
Comment (greg-fer) [inline doc/nrf/known_issues.rst]:
```suggestion
  If a ``c2d`` message is sent to the device while the device is in the LTE Power Saving Mode, the TCP connection will most likely be terminated by the server.
```

File: doc/nrf/known_issues.rst [hunk 3]
```diff
@@ -112,6 +107,21 @@ IRIS-2676: Missing support for FOTA on nRF Cloud
...
+  Known symptoms of this is frequent reconnections to cloud, that messages sent to Azure IoT Hub never arrives, and that FOTA images are downloaded twice.
+
+.. rst-class:: v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
+
+CIA-463: Wrong network mode parameter reported to cloud
+  The network mode string present in `deviceInfo` (nRF Cloud) and `dev` (Azure IoT Hub and AWS IoT) JSON objects that is reported to cloud might contain the wrong network modes.
```
Comment (greg-fer) [inline doc/nrf/known_issues.rst]:
```suggestion
  The network mode string present in ``deviceInfo`` (nRF Cloud) and ``dev`` (Azure IoT Hub and AWS IoT) JSON objects that is reported to cloud might contain wrong network modes.

**Review by greg-fer — CHANGES_REQUESTED** [review]
No workaround for these issues?

**Comment by simensrostad** [comment]
> No workaround for these issues?

Not currently I'm afraid.

**Pull request merged**